docs/Buttons/ButtonPress/index.md

```diff
@@ -0,0 +1,123 @@
+## ButtonPress
```

Comment:
This file needs to be added to the config.json [https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/master/config.json](url)